File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Invs/INV6000U00LayINV6000Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Invs
{
    public class INV6000U00LayINV6000Info
    {
        private String _pkinv6001;
        private String _magamMonth;
        private String _inputDate;
        private String _userName;
        private String _remark;
        private String _processTime;

        public String Pkinv6001
        {
            get { return this._pkinv6001; }
            set { this._pkinv6001 = value; }
        }

        public String MagamMonth
        {
            get { return this._magamMonth; }
            set { this._magamMonth = value; }
        }

        public String InputDate
        {
            get { return this._inputDate; }
            set { this._inputDate = value; }
        }

        public String UserName
        {
            get { return this._userName; }
            set { this._userName = value; }
        }

        public String Remark
        {
            get { return this._remark; }
            set { this._remark = value; }
        }

        public String ProcessTime
        {
            get { return this._processTime; }
            set { this._processTime = value; }
        }

        public INV6000U00LayINV6000Info() { }

        public INV6000U00LayINV6000Info(String pkinv6001, String magamMonth, String inputDate, String userName, String remark, String processTime)
        {
            this._pkinv6001 = pkinv6001;
            this._magamMonth = magamMonth;
            this._inputDate = inputDate;
            this._userName = userName;
            this._remark = remark;
            this._processTime = processTime;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/invs/INV6000U00LayINV6000Info.java
package nta.med.data.model.ihis.invs;

import java.util.Date;

import org.joda.time.DateTime;

public class INV6000U00LayINV6000Info {
	private Double pkinv6001       ;
	private String magamMonth     ;
	private String inputDate      ;
	private String userName       ;
	private String remark          ;
	private String processTime ;
	public INV6000U00LayINV6000Info(Double pkinv6001, String magamMonth, String inputDate, String userName,
			String remark, String processTime) {
		super();
		this.pkinv6001 = pkinv6001;
		this.magamMonth = magamMonth;
		this.inputDate = inputDate;
		this.userName = userName;
		this.remark = remark;
		this.processTime = processTime;
	}
	public Double getPkinv6001() {
		return pkinv6001;
	}
	public void setPkinv6001(Double pkinv6001) {
		this.pkinv6001 = pkinv6001;
	}
	public String getMagamMonth() {
		return magamMonth;
	}
	public void setMagamMonth(String magamMonth) {
		this.magamMonth = magamMonth;
	}
	public String getInputDate() {
		return inputDate;
	}
	public void setInputDate(String inputDate) {
		this.inputDate = inputDate;
	}
	public String getUserName() {
		return userName;
	}
	public void setUserName(String userName) {
		this.userName = userName;
	}
	public String getRemark() {
		return remark;
	}
	public void setRemark(String remark) {
		this.remark = remark;
	}
	public String getProcessTime() {
		return processTime;
	}
	public void setProcessTime(String processTime) {
		this.processTime = processTime;
	}
	
}
